### PR TITLE
POCPOC: Doctrine entity

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | try to delete db_structure :x
| Type?         |  improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | -

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: install-dev/data/db_structure.sql [hunk 1]
```diff
@@ -141,7 +39,7 @@ CREATE TABLE `PREFIX_cart` (
   `recyclable` tinyint(1) unsigned NOT NULL DEFAULT '1',
   `gift` tinyint(1) unsigned NOT NULL DEFAULT '0',
   `gift_message` text,
-  `mobile_theme` tinyint(1) NOT NULL DEFAULT '0',
+  `mobile_theme` tinyint(1) NOT NULL DEFAULT '0',git checkout
```
Comment (PrestaEdit) [inline install-dev/data/db_structure.sql]: ;-)

**Review by PrestaEdit — CHANGES_REQUESTED** [review]

File: install-dev/data/db_structure.sql (hunk 1 above)
Comment (aleeks) [inline install-dev/data/db_structure.sql]: ahah, i know, i'm testing on the flow!

Comment (aleeks) [inline install-dev/data/db_structure.sql]: thank you!

Comment (xBorderie) [inline install-dev/data/db_structure.sql]: Haha :)

Comment (Quetzacoalt91) [inline install-dev/data/db_structure.sql]: Nice catch

**Pull request closed without merging**